2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_setup.py:_flush():67] Configure stats pid to 118623
2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_setup.py:_flush():67] Loading settings from /home/hawk/.config/wandb/settings
2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_setup.py:_flush():67] Loading settings from /home/hawk/Desktop/Heavy-Clutter-Detection/Thesis/wandb/settings
2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-05-07 20:06:34,106 INFO    MainThread:118623 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /home/hawk/Desktop/Heavy-Clutter-Detection/Thesis/wandb/run-20250507_200634-2fk07r7p/logs/debug.log
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /home/hawk/Desktop/Heavy-Clutter-Detection/Thesis/wandb/run-20250507_200634-2fk07r7p/logs/debug-internal.log
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ed67ce72960>
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:init():781] calling init triggers
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'SNR': [10], 'CNR': [15], 'NU': [0.2, 0.5, 1.0], 'n_targets': 8, 'rand_n_targets': True, 'beta_end': 0.01, 'beta_start': 0.0001, 'batch_size': 16, 'num_epochs': 400, 'noise_steps': 2000, 'num_workers': 4, 'dataset_size': 50000, 'time_emb_dim': 256, 'learning_rate': 0.0001, 'scheduler_type': 'cosine', '_wandb': {}}
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:init():809] starting backend
2025-05-07 20:06:34,107 INFO    MainThread:118623 [wandb_init.py:init():813] sending inform_init request
2025-05-07 20:06:34,112 INFO    MainThread:118623 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-07 20:06:34,112 INFO    MainThread:118623 [wandb_init.py:init():823] backend started and connected
2025-05-07 20:06:34,113 INFO    MainThread:118623 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-05-07 20:06:34,116 INFO    MainThread:118623 [wandb_init.py:init():915] updated telemetry
2025-05-07 20:06:34,119 INFO    MainThread:118623 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-05-07 20:06:34,740 INFO    MainThread:118623 [wandb_init.py:init():1014] starting run threads in backend
2025-05-07 20:06:34,779 INFO    MainThread:118623 [wandb_run.py:_console_start():2454] atexit reg
2025-05-07 20:06:34,780 INFO    MainThread:118623 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-05-07 20:06:34,780 INFO    MainThread:118623 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-05-07 20:06:34,780 INFO    MainThread:118623 [wandb_run.py:_redirect():2394] Redirects installed.
2025-05-07 20:06:34,780 INFO    MainThread:118623 [wandb_init.py:init():1056] run started, returning control to user process
2025-05-07 20:06:34,780 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-07 20:06:34,780 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-07 20:06:34,783 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-07 20:11:53,849 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-07 20:11:53,849 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-07 20:11:53,852 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-07 20:11:54,315 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-07 20:11:54,315 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-07 20:11:54,319 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:01:22,016 WARNING MsgRouterThr:118623 [router.py:message_loop():63] [no run ID] message_loop has been closed
Traceback (most recent call last):
  File "/home/hawk/Desktop/Heavy-Clutter-Detection/finalDiffusion/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 27, in _read_message
    return self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hawk/Desktop/Heavy-Clutter-Detection/finalDiffusion/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 235, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hawk/Desktop/Heavy-Clutter-Detection/finalDiffusion/.venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 220, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/hawk/Desktop/Heavy-Clutter-Detection/finalDiffusion/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 56, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/hawk/Desktop/Heavy-Clutter-Detection/finalDiffusion/.venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 29, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-05-08 10:01:22,016 INFO    MsgRouterThr:118623 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-05-08 10:01:22,166 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:01:22,166 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:01:45,912 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:01:53,146 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:01:53,146 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:01:58,733 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:02:05,898 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:02:05,898 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:02:12,099 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:02:19,321 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:02:19,321 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:02:30,606 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:02:37,870 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:02:37,870 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:02:43,564 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:02:50,714 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:02:50,714 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:02:55,347 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:03:02,557 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:03:02,557 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:03:08,485 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:03:15,708 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:03:15,708 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:03:23,377 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:03:30,538 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:03:30,538 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:03:46,949 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:03:54,177 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:03:54,177 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:04:04,027 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:04:11,194 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:04:11,194 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:04:36,145 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:04:43,375 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:04:43,375 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-05-08 10:08:39,258 INFO    MainThread:118623 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-05-08 10:17:33,765 INFO    MainThread:118623 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-05-08 10:17:33,765 INFO    MainThread:118623 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
